### PR TITLE
documentation patch

### PR DESCRIPTION
**Pull request by sturkmen72**

### CODE REVIEW

**Comment by sturkmen72** [comment]
http://code.opencv.org/issues/4498
https://github.com/Itseez/opencv/issues/5071

File: modules/imgcodecs/include/opencv2/imgcodecs.hpp [hunk 1]
```diff
@@ -110,9 +110,11 @@ returns an empty matrix ( Mat::data==NULL ). Currently, the following file forma
 -   JPEG 2000 files - \*.jp2 (see the *Notes* section)
 -   Portable Network Graphics - \*.png (see the *Notes* section)
 -   WebP - \*.webp (see the *Notes* section)
--   Portable image format - \*.pbm, \*.pgm, \*.ppm (always supported)
+-   Portable image format - \*.pbm, \*.pgm, \*.ppm \*.pxm, \*.pnm (always supported)
```
Comment (vpisarev) [inline modules/imgcodecs/include/opencv2/imgcodecs.hpp]: do we really support pxm and pnm? I thought, pxm is a common term for pbm, pgm and ppm. And I don't know what pnm is

Comment (sturkmen72) [inline modules/imgcodecs/include/opencv2/imgcodecs.hpp]: https://github.com/Itseez/opencv/pull/4206

**Comment by mshabunin** [comment]
@sturkmen72, thank you for contribution!

I'd suggest to improve this area little more: it is better to move documentation for each flag to the corresponding enum and add reference to this enum to the function documentation. In case when enum is anonymous, you can assign it a name to be used for futher referencing.

Please, take a look at the documentation for function [cv::compare](http://docs.opencv.org/master/d2/de8/group__core__array.html#ga303cfb72acf8cbb36d884650c09a3a97), corresponding enum [cv::CmpTypes](http://docs.opencv.org/master/d2/de8/group__core__array.html#ga0cc47ff833d40b58ecbe1d609a53d784) and [same documentation in 2.4](http://docs.opencv.org/2.4/modules/core/doc/operations_on_arrays.html#compare).

__Note:__ the enum values and enums itself should be referred by adding _cv::_ prefix: `cv::WINDOW_NORMAL` or `cv::WindowFlags`. In this case doxygen will generate links to corresponding place.

**Pull request closed without merging**